src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy

```diff
@@ -48,12 +68,20 @@ class DebugSpec extends BaseSpec {
         bidRequest.ext.prebid.debug = test
         bidRequest.test = test
 
+        and: "Flash metrics"
+        flushMetrics(defaultPbsService)
+
         when: "PBS processes auction request"
         def response = defaultPbsService.sendAuctionRequest(bidRequest)
 
         then: "Response shouldn't contain ext.debug"
         assert !response.ext?.debug
 
+        and: "Debug metrics shouldn't be populated"
+        def metricsRequest = defaultPbsService.sendCollectedMetricsRequest()
+        assert !metricsRequest[AUCTION_REQUESTED_WITH_DEBUG_MODE_METRICS]
+        assert !metricsRequest.keySet().contains(ACCOUNT_METRICS_PREFIX_NAME)
```

Comment:
Please also verify the regular request metric. Make sure it is available and not duplicated